bnelson777/id.ly
# About screen icon updated

-Updated About screen icon and realigned github/pdx.edu icons

Testing steps
1. Open app and open dev mode
2. Access About page and ensure main icon shows new icon and the icons are aligned properly

![image-1](https://user-images.githubusercontent.com/32649350/40894066-bfb98fc2-675b-11e8-851b-980cb1cec114.jpg)
